src/transformers/models/deberta/modeling_tf_deberta.py

```diff
@@ -593,11 +593,10 @@ def call(
         else:
 
             def linear(w, b, x):
-                return tf.cond(
-                    b is not None,
-                    lambda: tf.matmul(x, w, transpose_b=True) + tf.transpose(b),
-                    lambda: tf.matmul(x, w, transpose_b=True),
-                )
+                out = tf.matmul(x, w, transpose_b=True)
+                if b is not None:
+                    out += b
```

Comment:
Is the transpose of b unnecessary here?

Reply:
I think it's needed! The "b" being transposed is the second matrix argument to `tf.matmul` (i.e. `w`), not the bias vector `b` in this code.

Reply:
Fixed!